powerscale/provider/snapshot_datasource_test.go

```diff
@@ -102,6 +119,18 @@ var SnapshotDataSourceConfig = `
 data "powerscale_snapshot" "test" {
   filter {
     path = "/ifs/tfacc_file_system_test"
+	state = "active"
```

Comment:
same for this as well, like creating a snapshot that exists on this path and then fetch the same using datasource